wolfcrypt/src/misc.c

```diff
@@ -546,6 +546,30 @@ WC_STATIC WC_INLINE byte ctSetLTE(int a, int b)
 }
 #endif
 
+#ifndef HAVE_FIPS
+/* This function used to be local to cmac.c. It was moved here because the
+ * AES-SIV code also needs it. cmac.c is part of the FIPS module, though, so we
+ * need to not compile this definition if using FIPS. */
+WC_STATIC WC_INLINE void ShiftAndXorRb(byte* out, byte* in, word32 sz)
```

Comment:
misc.c is for general functions like XOR buffers.
This seems algorithm specific.
I would like it in cmac.c as a local API or as a #define in a header file.

Reply:
That makes sense. I can make it `WOLFSSL_LOCAL` and part of cmac.c/h, and then it can be used in aes.c.